test/e2e/feature/feature.go

```diff
@@ -41,6 +41,10 @@ var (
 	// Marks tests that exercise the CBOR data format for serving or storage.
 	CBOR = framework.WithFeature(framework.ValidFeatures.Add("CBOR"))
 
+	// Owner: sig-node
+	// Marks test that exercise checkpointing of containers
```

Comment:
Generally it is good to add some comments on what is specific about the environment needed to run tests marked with this feature. We can start with simple migration, and update those later